src/main/fc/fc_msp.c

```diff
@@ -153,6 +156,9 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
     { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
     { BOXBLACKBOXERASE, "BLACKBOX ERASE (>30s)", 31 },
+    { BOXCAMERAWIFI, "CAMERA WIFI", 32},
+    { BOXCAMERAPOWER, "CAMERA POWER", 33},
+    { BOXCAMERACHANGEMODE, "CAMERA CHANGE MODE", 34 },
```

Comment:
I should note that we already take a similar approach for SERVOS.   I can imagine a scenario where different camera are supported and for which the 'BOXCAMERAx' functions will do things other than 'wifi, power and mode'.